### ISSUE TITLE
Table Columns Being Truncated

### ISSUE THREAD

**Issue by miguelwood** [iRon7/ConvertFrom-SourceTable#1]
In the attached example, the last column (AccessRights) is being truncated after the ConvertFrom-SourceTable is executed.

[ConvertFrom-SourceTable_ISSUE.txt](https://github.com/iRon7/ConvertFrom-SourceTable/files/2485803/ConvertFrom-SourceTable_ISSUE.txt)

**Comment by iRon7** [comment]
Hello Miguel,

Thanks for the feedback, I will look into this when I am back from holidays next week.

**Comment by iRon7** [comment]
Hello Miguel,

I have fixed the specific issue and improved the code in several ways to prevent losing floating data.
Besides, I have added a lot of tests to confirm any floating data is picked up where corresponding column can be determined.
Nevertheless, there are some indefinable situations like:
````
ConvertFrom-SourceTable '
C1    C2
--    --
11 1, 22
'
````
Where it is impossible to determine whether the data in the middle belongs to the (left-aligned) left (`C1`) column or the (right-aligned) right (`C2`) column.

Usually when more data is provided, it becomes obvious how the data is aligned, but in the case it is not and there are no other indications how the columns justified, it is now also possible to add a ruler to outline the columns:
````
ConvertFrom-SourceTable -Ruler '-- -----' '
C1    C2
--    --
11 1, 22
'
````
